README.md

```diff
@@ -103,6 +103,36 @@ with `T` being one of the following basic types:
 `bool` `char` `wchar_t` `uint8_t` `int16_t` `uint16_t` `int32_t` `uint32_t` `int64_t` `uint64_t` `float` `double`
 `long double`
 
+#### Enumerated Type
+EnumeratedTypes are a special kind of PrimitiveTypes. They are internally represented by a PrimitiveType, but only
+allows a user-defined subset of values.
+
+##### EnumerationType
```

Comment:
Why a subtitle? Why not directly EnumerationType once time?

Reply:
To be prepared when we add bitmasks.

Reply:
ok